packages/in_app_purchase/example/lib/main.dart

```diff
@@ -118,6 +117,31 @@ class _MyAppState extends State<MyApp> {
               'This app needs special configuration to run. Please see example/README.md for instructions.')));
     }
 
+    print('xyzzy about to query past purchases');
+    Map<String, PurchaseDetails> purchases = Map.fromEntries(((await connection
```

Comment:
Please put a comment here explaining that you should never actually do this and should always verify this data before trusting it, also using your own server if possible. I think it's reasonable for us to skip verifying purchases in this sample app since we couldn't really demo a server along with this, but we should still warn people that they don't want to follow this example as-is.

Reply:
done